WordPress/gutenberg
# Inspector: Improve handling of long words in post titles

Fixes https://github.com/WordPress/gutenberg/issues/62627.

## What?
Improves handling of long words in the UI. Specifically:

1. In the card title
2. In the slug button

### Before
https://github.com/WordPress/gutenberg/assets/846565/cefda298-1b46-410e-9505-5818929de9eb

### After
<img width="278" alt="Screenshot 2024-06-20 at 10 06 08" src="https://github.com/WordPress/gutenberg/assets/846565/475245a3-99f8-40cd-9015-b4a699e5ae3e">

## How?
* In the card title `hyphens: auto` is used to hyphenate long words.
* In the slug button `word-break: break-word` is used to break long words without hyphenating. This is to avoid confusion about the slug structure.

## Testing Instructions
1. Create a page with a title that contains a very long word.
2. Check the title in the Inspector and ensure the long word is hyphenated
3. Check the 'Link' button and ensure the long word is broken, and that the Inspector does not scroll horizontally.
